src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -158,3 +159,8 @@ void EquationInputArea::ReloadAvailableColors(bool isHighContrast)
         equationViewModel->LineColor = AvailableColors->GetAt(m_lastLineColorIndex);
     }
 }
+
+void CalculatorApp::EquationInputArea::UserControl_SizeChanged(Platform::Object ^ sender, Windows::UI::Xaml::SizeChangedEventArgs ^ e)
+{
+    EquationInputList->MaxHeight = ActualHeight - 44;
```

Comment:
This is the best solution I could come up with, would love to hear of any alternatives. The goal is to get the button to add an equation to be directly below the ListViewItems until it needs to scroll, at that point it becomes a sticky footer.

Reply:
I know a trick to do what you want! Let's chat about it tomorrow

Reply:
You need 2 things:
- Grid with 2 rows, one * (for the listview) and one auto (for the footer) and set `VerticalAlignment` to _Top.
- this Grid must be hosted in a Panel having a restricted Height (Grid, Border, not StackPanel or ScrollViewer)

```xaml
<Grid> <! -- parent with a restricted height -->
   <Grid VerticalAlignment="Top">
            <Grid.RowDefinitions>
                <RowDefinition Height="*" />
                <RowDefinition Height="auto" />
            </Grid.RowDefinitions>
            <ListView x:Name="MyList">
                <ListView.ItemTemplate>
                    <DataTemplate>
                        <TextBlock FontSize="60" Text="HELLO" />
                    </DataTemplate>
                </ListView.ItemTemplate>
            </ListView>
            <TextBlock Grid.Row="1" FontSize="60" Text="Footer" />
        </Grid>
</Grid>
```

Reply:
This is great, thank you!